Scalar.Common/Scalar.Common.csproj

```diff
@@ -8,13 +8,18 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="NuGet.CommandLine" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
     <PackageReference Include="NuGet.Commands" />
     <PackageReference Include="Microsoft.PowerShell.SDK" />
     <PackageReference Include="Microsoft.Windows.Compatibility" />
     <PackageReference Include="GitForWindows.GVFS.Installer" PrivateAssets="all" />
     <PackageReference Include="GitForMac.GVFS.Installer" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="$(PkgNuGet_CommandLine)\tools\nuget.exe" CopyToOutputDirectory="PreserveNewest" Visible="false" LinkBase="extbin" Condition="$(RuntimeIdentifier.StartsWith('win'))" />
```

Comment:
I see this is the only other use of "extbin". Is there a way to define this as a build-time variable that is also accessible in the C# code? I just want to make sure these two things never drift, especially if we add anything else into that directory.

Reply:
Done! Added it to part of the generated `ScalarConstants.g.cs` file, which comes from MSBuild variables.